LayoutTests/accessibility/mac/attributed-string-with-listitem-multiple-lines.html

```diff
@@ -18,7 +18,7 @@
 
     if (window.accessibilityController) {
         var listItem1 = accessibilityController.accessibleElementById("item1");
-        var p1 = listItem1.childAtIndex(0);
+        var p1 = listItem1.childAtIndex(1);
```

Comment:
Prior to this PR, we sometimes used to insert list markers as not the first child of list items, which is wrong. I fixed this bug as a side effect of this change, and thus the `<p>` is at index 1 (as it should be) rather than index 0.